### DISCUSSION TITLE
HTTP String und JSON Pfad beim Shelly 3 em

### DISCUSSION THREAD

**Discussion by iukbox** [hoylabs/OpenDTU-OnBattery#234] [Q&A] [answered]
Ein Bekannter hat einen Shelly 3 em und möchte den Powerlimiter einsetzen.
Der Shelly soll ja mit dem Aufruf http://ip_adresse/status seinen Status liefern. Im Browser funktioniert das auch.
![image](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/26631519/56eb126a-704f-4e97-b000-819f1566138d)
Wird die Adresse http://ip_adresse/status in der Konfiguration der DTU eingegeben kommt als Fehlermeldung Bad HTTP Code: 401.
Als JSON Pfad  ist emeters/total_power eingerichtet.
Hab ich was übersehen?

**Comment by bf4ed** [comment]
Schau was ich da auf den Bildern hab.
für die erste Faze (zweite /1/ und dritte /2/) - IP/emeter/0/
und unten bei Json path - power


https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/179

**Comment by helgeerbe** [comment, marked as answer]
Hm, von Interesse ist doch nur die `total_power`.

Versuche doch mal `http://admin:PASSWORD@shellyem3.fritz.box/status`. Password mit dem eigenen Passwort vom Shelly ersetzen.

Json path: `total_power`

Hint: total_power ist auf der gleichen Ebene wie emeters.

**Reply by iukbox** [reply]
Danke an beide.

**Comment by stefan123t** [comment]
Please double check the **Dynamic Power Limiter (DPL)** function in [OpenDTU-OnBattery](https://github.com/hoylabs/OpenDTU-OnBattery/).

I have described a [possible PM+DPL setup](https://github.com/tbnobody/OpenDTU/issues/272#issuecomment-2219644352) with a Shelly Pro3EM as **Power Meter (PM)** upstream in issue [#272](https://github.com/tbnobody/OpenDTU/issues/272). 
Just change -300W to -800W which allows your inverters to donate up to 800W to the power continuum.
